source/class/cv/parser/MetaParser.js

```diff
@@ -85,7 +85,9 @@ qx.Class.define('cv.parser.MetaParser', {
 
     parseIcons: function(elem) {
       cv.Config.configSettings.iconsFromConfig.push(this.__parseIconDefinition(elem));
-      qx.core.Init.getApplication().loadIcons();
+      cv.Config.configSettings.iconsFromConfig.forEach(function(icon) {
+        cv.IconHandler.getInstance().insert(icon.name, icon.uri, icon.type, icon.flavour, icon.color, icon.styling, icon.dynamic, icon.source);
+      }, this);
```

Comment:
This seems to be wrong here, it is called after every `icon-definition` entry. Icons should be loaded once after all icon-definitions have been parsed.